File: yolan/stdlib.js
var GLOBAL = (function () {
    return this;
})();
if (! GLOBAL . readline) {
    readline = (function () {
        importPackage(java . io);
        importPackage(java . lang);
        stream = BufferedReader(InputStreamReader(System["in"]));
        return function () {
            var line = stream . readLine();
            if (line !== null) {
                return "" + line;
            } else {
                return "";
            };
        };
    })();
};
var getch = (function () {
    var line = "";
    var pos;
    var newl = 0;
    return function () {
        pos = pos + 1;
        if (line[pos] !== undefined) {
            newl = 0;
            return line[pos];
        } else {
            pos = -1;
            line = readline();
            newl = newl + 1;
            if (newl > 10) {
                return "]";
            } else {
                return "\n";
            };
        };
    };
})();
function tail(list, n) {
    n = n || 1;
    return list.slice(n);
}
function put(obj, key, val) {
    obj[key] = val;
}
function map(fn, list) {
    return list.map(fn);
}
function array_join(list, sep) {
    return list.join(sep);
}
function array_push(list, elem) {
    return list.push(elem);
}
function string_contains(str, c) {
    return str.indexOf(c) != -1;
}
function array() {
    return Array.prototype.slice.call(arguments);
}
function strjoin() {
    return Array.prototype.slice.call(arguments).join("");
}
function len(obj) {
    return obj.length;
}
function get_type(obj) {
    return typeof(obj);
}

File: lightscript/.attic5/stdlib.py
import sys

def getch():
    c = sys.stdin.read(1)
    if c == '':
        return ']'
    else:
        return c
def array(*args):
    return [x for x in args]
def string_contains(str, elem):
    return elem in str
def array_push(arr, elem):
    arr.append(elem)
def array_join(arr, str):
    return str.join(arr)
false = False
true = True
def get_type(obj):
    return type(obj)
STRING_TYPE = get_type("")
def strjoin(*args):
    return "".join(args)
def tail(arr, n = 1):
    return arr[n:]
